Model/Export/ProductExportManager.php

```diff
@@ -860,4 +860,27 @@ private function getCheapestPrice($configurableProduct): float
 
         return !empty($simpleProductPrices) ? min($simpleProductPrices) : 0.00;
     }
+
+    /**
+     * @param ProductInterface $product
+     * @param string $productType
+     * @return string
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    private function isActive(ProductInterface $product, string $productType): string
+    {
+        $isActive = 'no';
+        if ($productType == 'configurable_assign') {
+            $parentIds = $this->configurableResource->getParentIdsByChild($product->getId());
+            if (count($parentIds) > 0) {
+                $parentId = reset($parentIds);
+                $parentProduct = $this->productRepository->getById($parentId);
+                $isActive = ($parentProduct->getStatus() == 1) ? 'yes' : 'no';
+            }
+        } else {
+            $isActive = ($product->getStatus() == 1) ? 'yes' : 'no';
+        }
```

Comment:
```suggestion
       $isActive = $product->getStatus() == 1;
        if ($isActive && $productType == 'configurable_assign') {
            $parentIds = $this->configurableResource->getParentIdsByChild($product->getId());
            if (count($parentIds) > 0) {
                $parentId = reset($parentIds);
                $parentProduct = $this->productRepository->getById($parentId);
                $isActive = $parentProduct->getStatus() == 1;
            }
        } 
        return    $isActive  ? 'yes' : 'no';
   
```

Isn't this only a deactivation condition?